### ISSUE TITLE
electric build shows error on build

### ISSUE THREAD

**Issue by mdelapenya** [wedeploy/wedeploy.com#349]
```
Error in plugin 'sass'
Message:
    src/styles/main.scss
Error: It's not clear which file to import for '@import "templates/root"'.
       Candidates:
         templates/_root.scss
         templates/root.scss
       Please delete or rename all but one of these files.
        on line 6 of src/styles/main.scss
>> @import "templates/root";
   ^
```

**Comment by jonnilundy** [comment]
@mdelapenya is this issue still occurring or was it fixed with #349?

**Comment by mdelapenya** [comment]
I cannot see that error any more. Closing :)

![image](https://media.giphy.com/media/AvMJCeu1EMmhG/giphy.gif)
